MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -859,6 +860,10 @@ private void setApiBaseUrl(@NonNull MapboxMapOptions options) {
     String apiBaseUrl = options.getApiBaseUrl();
     if (!TextUtils.isEmpty(apiBaseUrl)) {
       nativeMapView.setApiBaseUrl(apiBaseUrl);
+      TelemetryDefinition telemetry = Mapbox.getTelemetry();
+      if (telemetry != null) {
+        telemetry.setApiBaseUrl(apiBaseUrl);
+      }
```

Comment:
@chloekraw do we need to change the endpoint based on the user endpoint for tiles?

Reply:
No, for Maps SDK customers (not Vision/China), there is no relationship between the endpoint used for tiles and the endpoint used for events. This method should only configure the endpoint used for events.